README.md

```diff
@@ -41,7 +41,7 @@ No matter what category of app you have, you'll need your Intercom app id and th
 
 	- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
 		// Initialize Intercom
-    	[Intercom setApiKey:@"<#ios_sdk-...#>" forAppId:@"<#your-app-id#>"];
+    	[Intercom setApiKey:@"ios_sdk-<#your-api-key#>" forAppId:@"<#your-app-id#>"];
```

Comment:
Thanks for this. This could suggest that you need to combine `ios_sdk-` and you API key which is not the case. Perhaps `<#your_ios_api_key#>` would be clearer?